### PR TITLE
feat: new join icons and improved untangling

### PR DESCRIPTION
**Pull request by noahamac**
image

### CODE REVIEW

**Comment by noahamac** [comment]
blocked by: #22

File: src/d3-utils/joins.ts [hunk 1]
```diff
@@ -153,19 +153,21 @@ export function createLookmlJoinElement(svg: any, joinData: any, diagramDict: Di
     let disabled = Math.abs(aY - fY) < 50 ? true : false
 
     let joinedDegree = diagramDict.tableData[joinedTable][0].diagramDegree
-    let degreeMembers = diagramDict.yOrderLookup[joinedDegree].length
-    let stepPro = aY < fY && degreeMembers > 2
+    let degreeMembers = diagramDict.yOrderLookup[joinedDegree].length + 1
```
Comment (bryans99) [inline src/d3-utils/joins.ts:156]: Me thinks a lot of these lets should be consts

**Review by bryans99 — CHANGES_REQUESTED** [review]
consts instead of lets?

not branched off of main?

At a minimum change lets where you can just so you get into the habbit.

I will approve if you think its easier though. Not sure I understand the merge strategy here

File: src/d3-utils/joins.ts (hunk 1 above)
Comment (noahamac) [reply src/d3-utils/joins.ts:156]: You right. I'm gonna do a standalone PR for this

**Review by bryans99 — APPROVED** [review]
Approved on the proviso that lets/consts get fixed in another PR

**Pull request merged**